docs/iris/example_code/Meteorology/COP_maps.py

```diff
@@ -131,8 +131,7 @@ def main():
         width = left - first_plot_left + width
 
         # Add axes to the figure, to place the colour bar
-        colorbar_axes = fig.add_axes([first_plot_left, bottom + 0.07,
-                                      width, 0.03])
+        colorbar_axes = fig.add_axes([first_plot_left, 0.18, width, 0.03])
```

Comment:
The `bottom` returned from `get_position` on [line +127](https://github.com/SciTools/iris/blob/1d47d6969bf4106e852185734389d554bf3da599/docs/iris/example_code/Meteorology/COP_maps.py#L127) now actually aligns with the bottom of the plot axes - this wasn't the case for `matplotlib<3`.

The fix here is to explicitly set the bottom of the `colorbar_axes` to align with this example from `matplotlib=2.*`